### PR TITLE
[jesus_viloria] Pull request clase3

### PR DESCRIPTION
**Pull request by OneGuabaya**

### CODE REVIEW

File: Tarea/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (kenri) [inline Tarea/.editorconfig]: The name of your folder should be `firstname_lastname`

File: Tarea/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
```
Comment (kenri) [inline Tarea/.editorconfig]: You only need to add html, css and js, we don't need this file

**Review by kenri — CHANGES_REQUESTED** [review]

File: Tarea/.editorconfig (hunk 1 above)
Comment (OneGuabaya) [reply Tarea/.editorconfig]: This item was changed.

File: Jesús_Viloria/DOM y NPM [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Referencias Estudiadas:
```
Comment (kenri) [inline Jesús_Viloria/DOM y NPM]: The name of your folder should be `jesus_viloria`

**Review by kenri — CHANGES_REQUESTED** [review]

File: Tarea/.editorconfig (hunk 2 above)
Comment (OneGuabaya) [reply Tarea/.editorconfig]: This item was deleted.

PD: I modified my task, I just made a new commit with a basic js.

**Review by kenri — APPROVED** [review]

**Pull request merged**